.github/workflows/ci.yml

```diff
@@ -23,21 +25,23 @@ jobs:
           pdm run python -m pip install coverage pytest-github-actions-annotate-failures
       - run: pdm run robotpy coverage test
       - run: pdm run coverage xml
-      - uses: codecov/codecov-action@v3
+      - uses: codecov/codecov-action@v4
+        with:
+          use_oidc: true
```

Comment:
Codecov [no longer support](https://github.com/codecov/codecov-action#v4-release) tokenless uploads from GitHub Actions. This tells the action to auth with [GitHub Actions' OpenID Connect](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#enabling-openid-connect-for-your-cloud-provider) support.